libs/dh/shared/data-access-api/src/lib/generated/v1/model/search-results-dto.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * DataHub BFF
+ * Backend-for-frontend for DataHub
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+import { SearchResultItemDto } from './search-result-item-dto';
+
+
+export interface SearchResultsDto { 
```

Comment:
Suggestion: Could we prefix these types as search-results-* etc. is pretty generic to avoid possible conflicts in the future?